src/components/ButtonPanel.jsx

```diff
@@ -3,34 +3,34 @@ import Button from './Button';
 
 function ButtonPanel() {
   return (
-    <div>
-      <div>
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
+    <div className="btn-panel">
+      <div className="group-1">
+        <Button color="#e0e0e0" name="AC" />
+        <Button color="#e0e0e0" name="+/-" />
+        <Button color="#e0e0e0" name="%" />
         <Button name="/" />
       </div>
-      <div>
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
+      <div className="group-2">
+        <Button color="#e0e0e0" name="7" />
+        <Button color="#e0e0e0" name="8" />
+        <Button color="#e0e0e0" name="9" />
         <Button name="*" />
       </div>
-      <div>
-        <Button name="4" />
-        <Button name="5" />
-        <Button name="6" />
+      <div className="group-3">
+        <Button color="#e0e0e0" name="4" />
+        <Button color="#e0e0e0" name="5" />
+        <Button color="#e0e0e0" name="6" />
         <Button name="-" />
       </div>
-      <div>
-        <Button name="1" />
-        <Button name="2" />
-        <Button name="3" />
+      <div className="group-4">
+        <Button color="#e0e0e0" name="1" />
+        <Button color="#e0e0e0" name="2" />
+        <Button color="#e0e0e0" name="3" />
         <Button name="+" />
       </div>
-      <div>
-        <Button name="0" />
-        <Button name="." />
+      <div className="group-5">
+        <Button color="#e0e0e0" wide={true} name="0" />
```

Comment:
[Optional]
You don't need to add the boolean value. The following declaration format will do the trick,
```js
<Button color="#e0e0e0" wide name="0" />
```
For more info, you can check this [link](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md)